packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -205,23 +204,16 @@ const genDriverLoaderBackCompatConfig = (compatVersion: number): CompatConfig[]
 };
 
 const getNumberOfVersionsToGoBack = (numOfVersionsAboveV2Int1: number = 0): number => {
-	const [, semverInternal, prereleaseIndentifier] = fromInternalScheme(
-		codeVersion,
-		true,
-		true,
-	);
-	assert(semverInternal !== undefined, "Unexpected pkg version");
-
-	// Here we check if the release is an RC release. If so, we also need to account for internal releases when
+	const semverVersion = semver.parse(codeVersion);
+	assert(semverVersion !== null, "Unexpected pkg version");
```

Comment:
For easier troubleshooting if we hit this case. I was also going to suggest using a plain `throw` so this doesn't get converted into a hex code, but seems like we don't do this for test-utils or the previous code wouldn't still have a string...? Maybe still safer to use regular if + throw. 
```suggestion
	assert(semverVersion !== null, `Unexpected pkg version '${code Version}'`);
```